packages/test/test-end-to-end-tests/src/test/idCompressor.spec.ts

```diff
@@ -1085,15 +1071,14 @@ describeCompat("IdCompressor Summaries", "NoCompat", (getTestObjectProvider, com
 			ds2.context.id.length > 8,
 			"Data store's ID in attached container should not be short",
 		);
-		const dds2 = SharedDirectory.create(ds2.runtime);
-		assert(dds2.id.length > 8, "DDS's ID in attached container should not be short");
 	});
```

Comment:
Removed the above assert since the considition it tests for doesn't seem true. See `createChannel` code in `datastoreRuntime.ts`.